File: internal/db/model/group.go
package model

import (
	"database/sql"
	"time"
)

// Group 豆瓣小组模型
type Group struct {
	ID          string       `db:"id"`
	Name        string       `db:"name"`
	Alt         string       `db:"alt"`
	MemberCount int          `db:"member_count"`
	Created     time.Time    `db:"created"`
	CreatedAt   time.Time    `db:"created_at"`
	UpdatedAt   sql.NullTime `db:"updated_at"`
}

// TableName Group表名
func (g *Group) TableName() string {
	return "Group"
}

File: src/main/java/com/douban/bot/model/Group.java
package com.douban.bot.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Group {
    private Long id;
    private String groupId;
    private String name;
    private String alt;
    private Integer memberCount;
    private LocalDate created;  // 日期格式，不是日期时间
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
